scripts/mirror.test.js

```diff
@@ -0,0 +1,134 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import assert from 'node:assert/strict';
+
+import bcd from '../index.js';
+import getMatchingBrowserVersion from './mirror.js';
+
+describe('mirror', function () {
+  const match = (source_browser, source_version, dest_browser) => {
```

Comment:
Could we add a `describe('getMatchingBrowserVersion()'), () => {...}` around all this?